libr/core/cmd_anal.c

```diff
@@ -2621,6 +2625,8 @@ static void cmd_anal_esil(RCore *core, const char *input) {
 				}
 			} else eprintf ("esil.stats is empty. Run 'aei'\n");
 			break;
+		case '-':
+			sdb_foreach (esil->stats, sdb_del, esil);
```

Comment:
actually removing some keys wont make really any sense imho. the `aek` gives descriptive info about the emulation of the last emulated instruction. so you either delete it or keep it, right ?